packages/icons/src/align-content-center.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+import { IconProps } from "./types";
+
+export const AlignContentCenterIcon = React.forwardRef<
+  SVGSVGElement,
+  IconProps
+>(({ color = "currentColor", ...props }, forwardedRef) => {
+  return (
+    <svg
+      width="24"
```

Comment:
The new grid for icons is 16px, can we get them in that size now or you want to create a separate task? If separate please create the task now.

Reply:
ping

Reply:
do we need a todo with a link to an issue?